app/models/jargon.rb

```diff
@@ -21,14 +22,29 @@
 #
 
 class Jargon < ApplicationRecord
+  mount_uploader :image, JargonImageUploader
+
+  belongs_to :radio
+
   validates :name,
             presence: true,
             uniqueness: true
+
   validates :description, presence: true
+
   validates :radio_id, presence: true
+
   validates :major, presence: true
 
   bind_inum :major, JargonMajors
 
-  belongs_to :radio
+  after_initialize :set_image_default, if: :new_record?
+
+  private
+  def set_image_default
```

Comment:
__[RuboCop]__

Layout/IndentationWidth: Use 2 (not 0) spaces for rails indentation.


<!-- Comment by [SideCI](https://sideci.com) -->